fetch-in-react/my-app/src/Users.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import User from './User';
+import UserCard from './UserCard';
+import './Users.css';
+
+export default function Users() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [users, setUsers] = useState([]);
+  const [user, setUser] = useState();
+
+  /* your code here (hint: useEffect) */
+
+  useEffect(() => {
+    if (isLoading) {
+      const logJsonData = async () => {
+        try {
+          const response = await fetch('https://jsonplaceholder.typicode.com/users');
+          if (!response.ok) throw new Error(`Error Code: ${response.status} Error Message: It Broken`);
+          const dataJson = await response.json();
+          setUsers(dataJson);
+        } catch (err) {
+          setError(err);
+          console.error('Error:', err.message)
+        }
+      }
+      logJsonData();
+      setIsLoading(false);
```

Comment:
Putting it here works because you catch the errors and don't throw any. It could also be put in a `finally` block.